src/node/unocss/plugin.ts

```diff
@@ -0,0 +1,11 @@
+import type { UnoCSSConfig } from './types'
+import UnocssPlugin from '@unocss/vite'
+import { defineUnoCSSConfig } from './index'
+
+export function UnoCSSPlugin<Theme extends object = object>(
+  options?: UnoCSSConfig<Theme> | string
+) {
+  return typeof options === 'string'
+    ? UnocssPlugin<Theme>(options)
+    : UnocssPlugin<Theme>(defineUnoCSSConfig(options ?? {}))
```

Comment:
We should review this when using external configuration file (with no options), should work but we need to review it, I'm going to send a PR to UnoCSS to allow provide `resolveConfigResult` from Vite plugin when creating the context if this PR is ok: 
- https://github.com/unocss/unocss/blob/main/packages/vite/src/index.ts#L35
- https://github.com/unocss/unocss/blob/main/packages/shared-integration/src/context.ts#L15
- https://github.com/unocss/unocss/blob/main/packages/shared-integration/src/context.ts#L39